content/docs/guides/ha_scale/perf-scale.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Performance and Scalability"
+description: "Performance benchmark on control plane and data plane sidecar and scalability test of Open Service Mesh"
+type: docs
+---
+
+Open Service Mesh aims to create a high-performance service mesh while providing all the essential service mesh features. This document describes the test process and results we performance to evaluate the performance and scalability of OSM.
+
+## Testing environment
+
+This test is performed on a Kubernetes cluster provisioned and managed by [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/). The cluster has 100 Standard_DS2_v2 nodes (each node has 2 vCPUs and 7GB memory). It is ensured that there is only one OSM controller pod running with default resource quota during the test. OSM is configured in non-permissive traffic mode.
+
+## Process
+
+The testing mesh contains two types of service. One is a load generator (modified from [wrk2](https://github.com/giltene/wrk2)), which can send requests at specified rate, load balanced among all other services in the mesh. The load generator serves as the client side of a request. Throughout our test, there will be at most one load generator service. The other type of service is a simple echo application that replies with whatever it receives from an inbound request. It serves as the server side of a request. We can deploy multiple copies of the echo application in the mesh. In short, the mesh topology is a typical 1-to-N service architecture.
+
+<p align="center">
+  <img src="/docs/images/perf/mesh-topo.png" width="650"/>
+</p>
+
+During the test, the echo applications will be deployed all at the same time. Therefore, the control plane receives a load spike after the deployment starts. All the Envoy sidecars are connecting to the control plane simutaneously. After that, the control plane becomes much more idle. We deliberately create this load spike at test bootstrap time in order to explore the maximum scale that the mesh can handle. Each round of test will last for 15 minutes.
```

Comment:
This feels like it should be under the Scalability heading. I completely forgot that the services were going to be added in batches until I saw "peak time" on the graphs. It makes more sense to colocate process explanations with the specific tests they're describing